src/styles/main.scss

```diff
@@ -1,3 +1,83 @@
-body {
+@import "./abstracts/vars";
+@import "./abstracts/mixins";
+@import "./abstracts/functions";
+
+*,
+*::before,
+*::after {
   margin: 0;
+  padding: 0;
+  box-sizing: border-box;
+}
+
+html,
+body {
+  overflow-x: hidden;
+  height: 100vh;
+}
+
+body {
+  display: flex;
+  justify-content: center;
+  align-items: center;
+  background-image: url("../images/mesh-179.png");
+  background-repeat: no-repeat;
+  background-position: 0 0;
+  background-size: cover;
+  background-attachment: fixed;
+}
+
+.calendar {
+  display: flex;
+  flex-wrap: wrap;
+  width: calendarWidth($day-size, $day-gap, $calendar-pdg);
+  gap: $day-gap;
+  padding: $calendar-pdg;
+
+  $days:
+  "tue" 101px,
+  "wed" 202px,
+  "thu" 303px,
+  "fri" 404px,
+  "sat" 505px,
+  "sun" 606px;
```

Comment:
- Instead of putting here hard-coded values, use indexes of each day and later on you will be able to calculate the correct margin basing on indexes and other variables ;)
- this could be in variables file

Reply:
Thanks for the advise :) Btw... I thought that map will maby be more efficient and I wanted to show that I am familiar with that syntax too.